WcaOnRails/app/views/registrations/_edit_registrations_table_footer.html.erb

```diff
@@ -12,7 +12,11 @@
     <td></td>
     <td>
       <% if @competition.guests_enabled? %>
-        <%= registrations.sum :guests %>
+        <%# We need to convert `registrations` to an array here rather than computing the sum directly in the database.
+            The reason is that we incude `registration_events` table which results in having duplicated registrations
+            in the result set, thus the produced sum is too large.
+            See: https://github.com/cubing/worldcubeassociation.org/issues/678#issuecomment-222386567 %>
+        <%= registrations.to_a.sum(&:guests) %>
```

Comment:
I was thinking (but didn't express clearly) that the `to_a` should go in **edit_registrations.html.erb**

`<% registrations = @competition.registrations.public_send(status).includes(:user, :registration_events).to_a %>`

I think this might execute the query twice.